studio/components/interfaces/Settings/Logs/LogEventChart.tsx

```diff
@@ -27,6 +26,8 @@ const LogEventChart: React.FC<Props> = ({ data, onBarClick }) => {
     })
   }, [JSON.stringify(data)])
 
+  if (!data) return null
+
   return (
     <BarChart
```

Comment:
Sidenote: just wondering if its easy to use the BarChart component from UI library since this one's deprecated from ChartRenderer

Reply:
I reckon it's better to leave this one to another PR – this one is already pretty big 👀